mobile/src/components/share/Theme.jsx

```diff
@@ -0,0 +1,34 @@
+'use strict';
```

Comment:
would it make sense to move this file to src/utilities folder instead of components, and use .js instead of .jsx since it's not a component?